libs/community/langchain_community/callbacks/openai_info.py

```diff
@@ -1,10 +1,13 @@
 """Callback Handler that prints to std out."""
 import threading
-from typing import Any, Dict, List
+import os
+from typing import Any, Dict, List, Optional
 
 from langchain_core.callbacks import BaseCallbackHandler
 from langchain_core.outputs import LLMResult
 
+from prometheus_client import Counter, Gauge
```

Comment:
prometheus_client isn't a langchain-community required dependency so we should only import it within the functions that need it